### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by KetpuntoG**
Updating PR information to make the marketing process easier

### CODE REVIEW

**Review by DanielNino27 — CHANGES_REQUESTED** [review]
One small change

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -25,3 +25,25 @@ line and fill in the pull request template.
...
+* AUDIENCE — Who is this for?
+
+
+* CALLS TO ACTION — What do we want people seeing this to do?
+
+
+* KEYWORDS — What words should be included in this post?
+
+
+* Which of the following types of documents is most similar to your file?
```
Comment (DanielNino27) [inline .github/PULL_REQUEST_TEMPLATE.md:44]:
```suggestion
* Which of the following types of documentation is most similar to your file? 
```

**Comment by KetpuntoG** [comment]
> Sorry - one more that I didn't catch from the link

Thanks! I updated with some Cata's comments too

**Review by DanielNino27 — APPROVED** [review]
Looks good to me!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -25,3 +25,26 @@ line and fill in the pull request template.
...
+* AUDIENCE — Who is this for?
+
+  *Eg. Chemistry researchers, PL educators, beginners in quantum computing.*
+
+
+* KEYWORDS — What words should be included in the marketing post?
+
+
+* Which of the following types of documentation is most similar to your file? 
+(more details [here](https://www.notion.so/xanaduai/Different-kinds-of-documentation-69200645fe59442991c71f9e7d8a77f8))
```
Comment (CatalinaAlbornoz) [inline .github/PULL_REQUEST_TEMPLATE.md:46]: External contributors won't be able to see this page. Maybe add "learning-oriented", "task-oriented", etc below.

**Review by CatalinaAlbornoz — APPROVED** [review]
It looks good! I've just added a small suggestion.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (KetpuntoG) [reply .github/PULL_REQUEST_TEMPLATE.md:46]: We are aware of this, but after talking to Tom it has been decided to put it this way. Thanks for the review 🚀

**Pull request merged**